src/CoDa.jl

```diff
@@ -18,11 +19,19 @@ import Base: names, getproperty
 import LinearAlgebra: norm, dot, ⋅
 
 include("composition.jl")
+include("covariances.jl")
 include("matrices.jl")
 include("transforms.jl")
 include("utils.jl")
 
 export
+  # covariances
+  designmatrix,
+  compvarmatrix,
+  variationmatrix,
+  lrcovmatrix,
+  clrcovmatrix,
+
```

Comment:
Please move this exports to before the matrices exports. Try to follow the same order of the include statements.